.env

```diff
@@ -0,0 +1,3 @@
+PORT=8000
+API key:7qJnEPLg23tQ04I0yX3GUqtA7
```

Comment:
- Environment variables must `BE_LIKE_THIS` as per the Unix standard which you can find here: https://en.wikipedia.org/wiki/Environment_variable. No spaces, lowercase or punctuation (apart from an underscore) is allowed. There's nothing technically stopping you from doing this - but other programs reading this file (like `dotenv`) may not recognise it.
- The format for environment variable entries are `MY_KEY=MYVALUE` or `API_KEY=7qJnEPLg23tQ04I0yX3GUqtA7`. Line 1 is correct, but 2 and 3 are incorrect.